rafthttp/peer.go

```diff
@@ -162,10 +161,7 @@ func startPeer(tr http.RoundTripper, urls types.URLs, local, to, cid types.ID, r
 				select {
 				case writec <- m:
 				default:
-					p.r.ReportUnreachable(m.To)
-					if isMsgSnap(m) {
-						p.r.ReportSnapshot(m.To, raft.SnapshotFailure)
-					}
+					dropMessage(m, r)
```

Comment:
this function name is not descriptive. I think we do not try to drop message. Instead, we report the dropped message and try to re-propose if the message includes a proposal.